test/e2e/kubevirt.go

```diff
@@ -524,7 +499,104 @@ passwd:
 			}
 
 		}
-		fcosVM = func(idx int, labels map[string]string, butane string) (*kubevirtv1.VirtualMachine, error) {
+
+		addressesFromStatus = func(vmi *kubevirtv1.VirtualMachineInstance) func() ([]string, error) {
+			return func() ([]string, error) {
+				err := crClient.Get(context.TODO(), crclient.ObjectKeyFromObject(vmi), vmi)
+				if err != nil {
+					return nil, err
+				}
+				addresses := []string{}
+				for _, iface := range vmi.Status.Interfaces {
```

Comment:
I'm assuming we don't report ipv6 link-local addresses. Right ?

Reply:
nope